sampleSpecs/pizza.smithy

```diff
@@ -336,3 +336,29 @@ structure EchoBody {
   @length(min: 10)
   data: String
 }
+
+@readonly
+@http(method: "GET", uri: "/get-defaults", code: 200)
+operation RenderDefaults {
+  output: GetDefault
+}
+
+list AList {
+  member: String
+}
+
+map AMap {
+  key: String
+  value: Integer
+}
+
+structure GetDefault {
```

Comment:
there should be a test exhibiting `@internal` too.

Reply:
The test [here](https://github.com/disneystreaming/smithy4s/pull/685/files#diff-350b86e9f4f0b2ebf6cbc04aec8cc0d8e3f66948796bd79e499eb7a43bd3d0d5R364-R370) does that: it renders a `GetDefault` structure which has a `someOtherInternalString` member with a default with an `Internal` trait on it

The test shows that it doesn't serialize it (server side)